manage.py

```diff
@@ -73,7 +73,8 @@ def _should_patch_gevent(args, gevent_commands):
 
 
 def set_default_settings_path(argv):
-    if len(argv) > 1 and argv[1] == 'test':
+    if len(argv) > 1 and argv[1] == 'test' or os.environ.get('CCHQ_TESTING') == '1':
+        os.environ.setdefault('CCHQ_TESTING', '1')
```

Comment:
What's the intend behind the name `CCHQ_TESTING`?
Is it to signify that this should currently run in test environment? how about `TEST_ENV` Or `ENV_TEST` or something better?

Reply:
It's a variable that puts the CommCareHQ management commands in test mode (use testsettings.py and run against test databases). I would have used `TESTING`, but that seemed a bit too generic (risk of being set for some other purpose); I wanted a name specific to HQ. I don't expect this to be used in HQ production environments. So far I've only used it for one-off maintenance of my local test database.